index.js

```diff
@@ -174,6 +174,11 @@ module.exports = class Localdrive {
     const { keyname, filename } = keyResolve(this.root, key)
     return new FileWriteStream(filename, keyname, this, opts)
   }
+
+  checkout () {
+    // Dummy, for compatibility (no versioning with localdrive)
```

Comment:
Just remove the comment like `batch()`, they should be together above in the class